src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -11,16 +11,57 @@ namespace Microsoft.CodeAnalysis.Sarif.Visitors
 {
     public class SarifVersionOneToCurrentVisitor : SarifRewritingVisitorVersionOne
     {
-        private static string[] TextMimeTypes = { "text/plain",
-                                                  "text/x-c",
-                                                  "text/x-h",
-                                                  "text/x-csharp",
-                                                  "text/x-java-source",
-                                                  "text/javascript",
-                                                  "text/ecmascript",
-                                                  "text/html",
-                                                  "text/xml" };
-            
+        #region Text MIM# types
+        private static HashSet<string> s_TextMimeTypes = new HashSet<string>()
+        {
+            "application/ecmascript",
+            "application/java-archive",
+            "application/javascript",
+            "application/java-serialized-object",
+            "application/java-vm",
+            "application/json",
+            "application/onenote",
+            "application/pdf",
```

Comment:
Some of these are not text format. Just keep the ones that we know are text:
- ecmascript
- javascript
- rss+xml
- rtf
- typescript
- x-csh
- xhtml+xml
- xml
- x-sh

Reply:
Done

Reply:
@EasyRhin0 I'm so sorry! `application\json` is text -- given our context, I don't know how I missed that!